rosdep/base.yaml

```
@@ -114,6 +114,7 @@ aravis:
   nixos: [aravis]
   ubuntu:
     '*': [libaravis-0.6-0, aravis-tools]
+    'jammy': [libaravis-0.8-0, aravis-tools]
     bionic: null
```

Seeing as 0.6.0 is only available for Focal, I would rewrite this as,

```suggestion
    '*': [libaravis-0.8-0, aravis-tools]
    bionic: null
    focal: [libaravis-0.6-0, aravis-tools]
```